routes/Login.py

```diff
@@ -8,5 +8,9 @@ def __init__(self, path, methods) -> None:
     def getResponse(self, request: Request):
         if request.req_type == "POST":
             return Route.buildResponse()
+        elif request.req_type == "GET":
+            f = open("./public/login.html", "rb")
+            file = f.read()
+            file_length = str(len(file))
 
-        
\ No newline at end of file
+            return Route.buildResponse(200,{"Content-Type":"text/html","X-Content-Type-Options":"nosniff","Content-Length":file_length},file)
```

Comment:
delete the "X-Content-Type-Options" and "Content-Length", response builder adds those by itself, no need for clutter here